compiler/p/codegen/OMRCodeGenerator.hpp

```diff
@@ -172,6 +172,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
    bool is64BitProcessor();
 
+   virtual bool getSupportsIbyteswap();
```

Comment:
This isn't your doing, but I don't think this needs to be virtual here or in the base class. Since this class is part of an extensible hierarchy you can override `getSupportsIbyteswap()` in the sub class without it being a virtual method (unless I am mistaken?).